### ISSUE TITLE
Control maximum output size

### ISSUE THREAD

**Issue by blootsvoets** [RADAR-CNS/RADAR-RestApi#107]
The server may now crash on too large requests, due to a `OutOfMemoryException`. The server should pre-evaluate requests and refuse too large intervals.

This may use the following code: https://github.com/RADAR-base/RADAR-RestApi/blob/0b855a1781d37036f8285134d3b370f62c7177e7/src/main/java/org/radarcns/service/DataSetService.java#L368

**Comment by yatharthranjan** [comment]
Error logs 

```
ERROR - [500] http://10.200.100.170:80/api/data/Radar-test%20/b7e5a48c-c16d-4b11-845b-b2f2626b4e75/cf8f031c-032b-4e40-952b-5cde7f3d0be9/ANDROID_PHONE_v1_ACCELEROMETER/MEDIAN [org.radarcns.webapp.exception.UncaughtExceptionMapper:32] 
com.mongodb.MongoQueryException: Query failed with error code 96 and error message 'Executor error during find command: OperationFailed: Sort operation used more than the maximum 33554432 bytes of RAM. Add an index, or specify a smaller limit.' on server hotstorage:27017
```

**Comment by blootsvoets** [comment]
Should be fixed with #112.
